src/Domain/Structure/ExcisionSphere.hpp

```diff
@@ -0,0 +1,81 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Defines class ExcisionSphere.
+
+#pragma once
+
+#include <array>
+#include <cstddef>
+#include <iosfwd>
+#include <limits>
+
+#include "Utilities/MakeArray.hpp"
+
+/// \cond
+namespace PUP {
+class er;
+}  // namespace PUP
+/// \endcond
+
+/// \ingroup ComputationalDomainGroup
+/// The excision sphere information of a computational domain.
+///
+/// \tparam VolumeDim the volume dimension.
+template <size_t VolumeDim>
+class ExcisionSphere {
```

Comment:
I considered it could be used for VolumeDim = 2 in the case where we have a Disk with an excised region - in 1D the excised region would be an interval, and we would have a disconnected domain - it makes sense mathematically (i think) but it's not anything I've seen used in SpECTRE

Reply:
I'm not sure disconnected domains are useful, but you're right that 2D could be, so let's leave it general.